frontend/src/metabase/admin/datamodel/components/revisions/RevisionHistory.jsx

```diff
@@ -20,7 +20,7 @@ class RevisionHistory extends Component {
   };
 
   render() {
-    const { object, objectType, revisions, table, user } = this.props;
```

Comment:
`objectType` was undefined.
But in any way, this component is now only used for segments in `master` so it's safe to remove this and to fetchSegments directly.